guides/source/active_record_validations.md

```diff
@@ -104,17 +108,18 @@ irb> p.new_record?
 ```
 
 Creating and saving a new record will send an SQL `INSERT` operation to the
```

Comment:
"Creating" is somewhat confusing here to me. Does it mean instantiation an new object or calling `Person.create`?
As the example above saves a new record, can we remove it?
```suggestion
Saving a new record will send an SQL `INSERT` operation to the
```

Reply:
Yes makes sense!